frontend/src/components/DecayInformations/CollapseLinksList.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div class="collapse-links-list">
+    <div class="d-flex">
+      <div class="text-center pb-3 gradido-max-width">
+        <b-icon icon="droplet-half" height="12" class="mb-2" />
+        <b>{{ $t('links-list.header') }}</b>
```

Comment:
if you do the dropdown content in another PR this should open empty or not open at all

```suggestion
```

Reply:
i made this yesterday quite quickly and it should only serve as a box where the correct info will be in the future. the icon is not used at this point. but i overlooked it.